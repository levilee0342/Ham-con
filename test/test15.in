-5181
-3658
-25781
-14099
