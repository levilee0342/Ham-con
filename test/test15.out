-3658
-25781